### DISCUSSION TITLE
How connect to user RDP session to PC with agent

### DISCUSSION THREAD

**Discussion by Sandteufel** [Ylianst/MeshCentral#6629] [General]
Hi!
Some my users worn not at office. THey use RDP connection to they worklstations (win10 or linux) If user are directly work on PC I can connect to his session and help him with problems/ But if he connect to PC remoutly (using RDP) I didn't see this connection? and can't connect to his session.

What  can I do to reduse this problem?

**Comment by si458** [comment]
generally when you click the `Connect` option, it would show a pop up saying which session u want to connect too
![image](https://github.com/user-attachments/assets/286dcbac-38d9-463e-b4e0-5aa5cc16499d)
